modules/backup-etcd.adoc

```diff
@@ -21,6 +21,15 @@ You should only save a backup from a single master host. You do not need a backu
 +
 ----
 $ sudo /usr/local/bin/etcd-snapshot-backup.sh ./assets/backup
+1bf371f1b5a483927cd01bb593b0e12cff406eb8d7d0acf4ab079c36a0abd3f7
+etcdctl version: 3.3.18
+API version: 3.3
+found latest kube-apiserver-pod: /etc/kubernetes/static-pod-resources/kube-apiserver-pod-7
+found latest kube-controller-manager-pod: /etc/kubernetes/static-pod-resources/kube-controller-manager-pod-8
+found latest kube-scheduler-pod: /etc/kubernetes/static-pod-resources/kube-scheduler-pod-6
+found latest etcd-pod: /etc/kubernetes/static-pod-resources/etcd-pod-2
+Snapshot saved at /var/home/core/assets/backup/snapshot_2020-03-18_220218.db
```

Comment:
@retroflexer I replaced the instances of `myback99` with `assets/backup` in this output, as that's what we use in this example command. Please make sure that this is correct.

Reply:
OK. That's fine. Keep it consistent.